server/config/validation/create-courtroom.spec.js

```diff
@@ -0,0 +1,41 @@
+(function() {
+  'use strict';
+  var validate = require('validate.js')
+    , validator = require('./create-courtroom')
+    , validatorResult = null;
+
+  describe('Administration - Create courtroom validators:', function() {
+
+    beforeEach(function() {
+      validatorResult = null;
+    });
```

Comment:
we can now just scope the `validatorResult` to each test case.

```js
it('should test something', function() {
  // ... some code ...

  const validatorResult = validate();

  // ... more code ...
});
```

Reply:
Updated.